tests/transforms/test_convert_rois.py

```diff
@@ -143,6 +125,7 @@ def test_binarize_and_convert_rois(s2p_stat_fixture, ophys_movie_fixture,
         'motion_corrected_video': str(movie_path),
         'motion_correction_values': str(motion_path),
         'output_json': str(output_path),
+        'npixel_threshold': 1
```

Comment:
I feel like it might be better to have a separate test for the `_small_size_exclusion()`. Although this test well have CodeCov report 'covered lines', the test cases don't really appear to be seriously testing the `_small_size_exclusion()` functionality.